backend/src/utils/resourceUtils.ts

```diff
@@ -562,8 +562,8 @@ export const initializeWatchedResources = (fastify: KubeFastifyInstance): void =
 };
 
 export const getDashboardConfig = (): DashboardConfig => {
-  const config = dashboardConfigWatcher.getResources()?.[0];
-  return _.merge({}, blankDashboardCR, config); // merge with blank CR to prevent any missing values
+  return dashboardConfigWatcher.getResources()?.[0];
+  // return _.merge({}, blankDashboardCR, config); // merge with blank CR to prevent any missing values
```

Comment:
Out of curiosity, why did you comment rather than delete it?

Reply:
Ope. Forgot to delete, no reason to comment.